app/controllers/data_sources_controller.rb

```diff
@@ -32,7 +32,7 @@ def toggle_reload_nightly
   end
 
   def new
-    @series = Series.find data_source_params[:series_id]
+    @series = Series.find params[:series_id]
```

Comment:
And this change is to fix https://uhero-analytics.atlassian.net/browse/UA-1090

Reply:
Thanks for catching this bug!